src/status_im/contexts/profile/contact/header/view.cljs

```diff
@@ -31,4 +34,14 @@
       {:title            full-name
        :description      :text
        :description-text bio
-       :emoji-dash       emoji-hash}]]))
+       :emoji-dash       emoji-hash}]
+     [rn/view {:style style/button-wrapper}
+      [quo/button
+       {:on-press  #(rf/dispatch [:show-bottom-sheet
+                                  {:content (fn [] [contact-request/view
+                                                    {:customization-color customization-color
+                                                     :profile-picture     profile-picture
+                                                     :full-name           full-name
+                                                     :public-key          public-key}])}])
+        :icon-left :i/add-user}
+       (i18n/label :t/send-contact-request)]]]))
```

Comment:
I'm wondering if I should subscribe directly to these values from inside the `contact-request/view` component, or just pass them along like I'm doing here. Thoughts?

Reply:
generally avoid passing subscription values along and prefer to subscribe inside the component using the values.